src/index.js

```diff
@@ -53,6 +53,6 @@ app.use((err, req, res, next) => {
 app.get('/', (req, res) => res.json({ status: res.statusCode, message: 'Welcome to Wolfsbane' }));
 
```

Comment:
This file is not in sync with that on develop branch. Please Rebase

Reply:
I just checked it now, it seems to me that they are in sync, is there anything specific in that line that you will like to point out??

Reply:
My bad. I saw just part of it